Formula/zion.rb

```diff
@@ -0,0 +1,21 @@
+class Zion < Formula
+  desc "Provides E2EE gateway to create secure communication"
+  homepage "https://nullby1e.github.io/zion/"
+  url "https://github.com/a1eaiactaest/zion-formula/releases/download/0.0.2/zion-0.0.2.tar.gz"
+  version "0.0.2"
+  sha256 "11d089faa1332743d4f02e199e244962fb770857fe6e2f577c2fe152f8f94e41"
+  license "WTFPL"
+
+  depends_on "go" => :build
+  depends_on "tor" => :build
+  depends_on "unzip" => :build
+
+  def install
+    system "./configure"
+    bin.install "zion-gateway"
+  end
+
+  test do
+    pipe_output("#{bin}/zion-gateway -h", 0)
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.